sdk/test/Services/ApplicationDiscoveryService/IntegrationTests/ApplicationDiscoveryServiceTests.cs

```diff
@@ -1,42 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-using AWSSDK_DotNet.IntegrationTests.Utils;
-
-using Amazon.Runtime;
-using Amazon.ApplicationDiscoveryService;
-using Amazon.ApplicationDiscoveryService.Model;
-using Amazon;
-using System.IO;
-using System.Text;
-using Amazon.APIGateway.Model;
-using Amazon.APIGateway;
-
-namespace AWSSDK_DotNet.IntegrationTests.Tests
```

Comment:
"For services without hand-written code, I'll delete the tests" - are you using a script to identify this or something else

Reply:
I did, but I don't have it saved it anywhere so will need to create it again :)

In my design doc there's a (somewhat outdated) list in the appendix, it's essentially services that have integration tests but nothing in their `Custom` folder (or just a `GlobalSuppressions` file).